docs/content/docs/recipes/teams.md

```diff
@@ -147,14 +147,14 @@ Please make sure to replace `<your tab name>` with the acutal tab name from your
 
 {{< callout warning >}}
 
-Since we are using the content page URL to store the container ID, this record will be removed if the Teams tab is deleted.
+Since content page URL is used to store the container ID, this record will be removed if the Teams tab is deleted.
 Additionally, every content page can only support one container ID.
 
 {{< /callout >}}
 
 ### Refactor content page to reflect Fluid application
 
-Open the file `src/client/<your tab name>/<your tab name>.tsx` in your code editor. A standard Fluid powered application consists of view and Fluid data component. For modularity and readability sake, let's just focus on getting/loading the Fluid container and leave all the Fluid related interactions in a React component.
+Open the file `src/client/<your tab name>/<your tab name>.tsx` in your code editor. A standard Fluid powered application consists of view and Fluid data component. Let's just focus on getting/loading the Fluid container and leave all the Fluid related interactions in a React component.
```

Comment:
```suggestion
Open the file `src/client/<your tab name>/<your tab name>.tsx` in your code editor. A typical Fluid-powered application consists of a view and a Fluid data component. Let's just focus on getting/loading the Fluid container and leave all the Fluid related interactions in a React component.
```

"data component" is a new term. What are you referring to?

Reply:
changed it to `fluid data structure`